src/time_tracker/invoices/handlers.clj

```diff
@@ -105,8 +119,11 @@
                       connection
                       invoice-id)]
       (do
-        (log/debug {:event "invoice-received" :data invoice})
-        (res/response invoice))
+        (log/debug {:event "invoice-received" :data invoice})    
+        (let [content-type (:content-type request)]
```

Comment:
You can also de-structure the `request` to get content type directly or use the first `let` block
However if you use the first let block you would have to use `do` for doing the logging.